vllm/envs.py

```diff
@@ -295,7 +295,7 @@ def maybe_convert_int(value: Optional[str]) -> Optional[int]:
 
     # this is used for configuring the default logging level
     "VLLM_LOGGING_LEVEL":
-    lambda: os.getenv("VLLM_LOGGING_LEVEL", "INFO"),
+    lambda: os.getenv("VLLM_LOGGING_LEVEL", "INFO").upper(),
```

Comment:
wondering why do we only have special handling on this env var :-)

@youkaichao, any thoughts?

Reply:
are there other logging level env vars?

Code interpreting this one expects uppercase only.

Reply:
we do have something similar:

```python
    "VLLM_USE_MODELSCOPE":
    lambda: os.environ.get("VLLM_USE_MODELSCOPE", "False").lower() == "true",
```

so this pr looks good to me.